spec/spec_helper.rb

```diff
@@ -94,4 +96,7 @@
   # as the one that triggered the failure.
   Kernel.srand config.seed
 =end
+
+  # Include our new authentication helper in all of our feature specs
+  config.include Helpers::Authentication, type: :feature
```

Comment:
normally if we are going to include these kinds of helpers we do it inside the `rails_helper` and not the `spec_helper`, could you please move this there